src/main.rs

```diff
@@ -1,16 +1,10 @@
-#![recursion_limit = "128"]
```

Comment:
Any reason this was added? Just for the `js` macro? Has it been fixed?

Reply:
Errr I think I just tried removing it and forgot to re-add it in when it worked. I'll double check.

Reply:
I think stdweb 0.4.11's change to a proc macro might have made it unnecessary?

I'll stick it in a different PR though